Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -0,0 +1,320 @@
+
+    return alert_as_dict
+
+
+def enrich_incident(reco_client: RecoClient, single_incident: Dict[str, Any]) -> Dict[str, Any]:
+    alert_as_dict = parse_table_row_to_dict(single_incident.get("cells", {}))
+    if RECO_INCIDENT_ID_FIELD in alert_as_dict.keys():
+        incident_id: str = str(alert_as_dict[RECO_INCIDENT_ID_FIELD])
+        alert_as_dict["assets"] = reco_client.get_incidents_assets(incident_id)
+
+    return {
+        "name": alert_as_dict.get("incident_description", ""),
+        "occurred": alert_as_dict.get("event_time", ""),
+        "dbotMirrorId": alert_as_dict.get("incident_id", ""),
+        "rawJSON": json.dumps(alert_as_dict),
+        "severity": map_reco_score_to_demisto_score(
+            reco_score=alert_as_dict.get("risk_level", DEMISTO_INFORMATIONAL)
+        ),
+    }
+
+
+def map_reco_score_to_demisto_score(
+        reco_score: int,
+) -> Union[int, float]:  # pylint: disable=E1136
+    # demisto_unknown = 0  (commented because of linter issues)
+    demisto_informational = 0.5
+    # demisto_low = 1  (commented because of linter issues)
+    demisto_medium = 2
+    demisto_high = 3
+    demisto_critical = 4
+
+    # LHS is Orca score
+    MAPPING = {
+        40: demisto_critical,
+        30: demisto_high,
+        20: demisto_medium,
+        10: demisto_informational,
+    }
+
+    return MAPPING[reco_score]
+
+
+def parse_incidents_objects(reco_client: RecoClient, incidents_raw: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
+    demisto.info("parse_incidents_objects enter")
+    incidents = []
+    for single_incident in incidents_raw:
+        incident = enrich_incident(reco_client, single_incident=single_incident)
+        incidents.append(incident)
+
+    demisto.info(f"get_incidents_from_alerts: Got {len(incidents)} incidents")
+    return incidents
+
+
+def fetch_incidents(
+        reco_client: RecoClient,
+        last_run: Dict[str, Any],
+        max_fetch: int,
+        risk_level: Optional[int] = None,
+        source: Optional[str] = None,
+        before: Optional[datetime] = None,
+        after: Optional[datetime] = None,
+) -> Tuple[Dict[str, Any], List[Dict[str, Any]]]:
+    demisto.info(f"fetch-incidents called {max_fetch=}")
+    next_run = {}
+    last_run_time = last_run.get("lastRun", None)
+    if last_run_time is not None:
+        after = dateutil.parser.parse(last_run_time)
+
+    incidents_raw = reco_client.get_incidents(
+        risk_level=risk_level,
+        source=source,
+        before=before,
+        after=after,
+        limit=max_fetch,
+    )
+    incidents = parse_incidents_objects(reco_client, incidents_raw)
+    incidents = [
+        incident
+        for incident in incidents
+        if incident.get("severity", 0) > DEMISTO_INFORMATIONAL
+    ]  # type: ignore
+
+    incidents_sorted = sorted(incidents, key=lambda k: k["occurred"])
+    next_run["lastRun"] = incidents_sorted[-1]["occurred"] if incidents_sorted else last_run_time
```

Comment:
The issue now might be duplicate incidents, because the first incident in the current fetch could be the last incident in the previous fetch because we are fetching from its occurred time.

The possible solution for this is to save the ids of the fetched incidents in the `lastRun` object and filter them out in the next fetch if they were fetched again.